### ISSUE TITLE
Harivainen cover image

### ISSUE THREAD

**Issue by Lincsand** [caseyg/knutepunkt2017#179]
Big black box on intro?  Picture?

**Comment by Martinemonster** [comment]
Picture and attribution to be uploaded. You find it here: https://drive.google.com/drive/folders/0B-3vI2IV4AW9R1UtZXZ2c0FQRHc

**Comment by caseyg** [comment]
We were actually thinking that the black square stays empty for this one — if that's ok @Lincsand @Martinemonster? We like the idea of a representing the story about the knutebooks with an illustration of an image-less book cover. Alternately we're thinking about making a book cover with recursive images of all the previous book covers.

**Comment by caseyg** [comment]
Related story: #57

**Comment by Martinemonster** [comment]
Let's go with the recursive images of all the previous book covers. The black cover looks like something's missing.

**Comment by Lincsand** [comment]
Uhm, do we have permission to use them in that case?

**Comment by Martinemonster** [comment]
Good point. We don't have time to get permission from all the past artists. But the black square looks like something's missing.

**Comment by caseyg** [comment]
Thanks, both! We'll think on it.
